### DISCUSSION TITLE
Customize learning rates for modules in architecture

### DISCUSSION THREAD

**Discussion by aditya9710** [open-mmlab/mmdetection3d#2016] [Q&A]
I am running the PointPillars architecture on KITTI dataset. This architecture has mainly three modules, a Feature Extractor, a 2D CNN Backbone, and a Detection Head. I want to use different learning rates for these modules. How can I do so with the mmdetection3d framework? Any examples would be greatly appreciated.

**Comment by Xiangxu-0103** [comment]
You can refer to this [tutorial](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/optim_wrapper.html#id7) for usage.
